cityofaustin/atd-product
# 10721 products page

Fixes cityofaustin/atd-data-tech/issues/10721

Steps to test:

[Netlify link](https://deploy-preview-53--atd-product.netlify.app/products)

1. Navigate to the Products page, there should be 4 cards per row on a laptop/desktop view
2. Scale down breakpoints, tablet view will show 2 cards per row, mobile will show 1 card per row
